src/components/Drawer/EditReportDrawer/EditReportDrawerContent.tsx

```diff
@@ -69,6 +68,45 @@ const StatusValueSelect: React.FC = () => {
   );
 };
 
+const EditDescription: React.FC = () => {
+  const { t } = useTranslation();
+
+  const dispatch = useDispatch();
+  const reportData = useSelector((state: IRootState) => state.report.data);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = e.target.value;
+    dispatch(
+      updateAddReportData({
+        target: {
+          ...reportData.target,
+          description: newValue,
+        },
+      }),
+    );
```

Comment:
Same as above, pass value directly

Reply:
I removed the newValue parameter, and passed e.target.value to description directly.